src/pages/index.js

```diff
@@ -30,55 +36,202 @@ const addCardPopup = new PopupWithForm(
   "#profile__add-card-modal",
   handleAddCardSubmit
 );
+const editAvatarPopup = new PopupWithForm(
+  "#profile__avatar-modal",
+  handleAvatarSubmit
+);
 const previewImagePopup = new PopupWithImage("#card_modal");
+const trashConfirmPopup = new PopupWithConfirm("#trashcan-modal");
+
+//Modal Event Listeners
+profileEditPopup.setEventListeners();
+addCardPopup.setEventListeners();
+editAvatarPopup.setEventListeners();
+trashConfirmPopup.setEventListeners();
+previewImagePopup.setEventListeners();
 
 // Section
-const cardSection = new Section(
-  { items: initialCards, renderer },
-  ".cards__list"
-);
-cardSection.renderItems();
+const cardSection = new Section({ items: [], renderer }, ".cards__list");
+
+// Validators
+const profileEditForm = document.querySelector("#profileEditForm");
+const addCardForm = document.querySelector("#addcard__form");
+const avatarForm = document.querySelector("#changeprofile_form");
 
-// Form validators
 const profileEditFormValidator = new FormValidator(config, profileEditForm);
-const addCardFormValidator = new FormValidator(config, addCardForm);
 profileEditFormValidator.enableValidation();
+const addCardFormValidator = new FormValidator(config, addCardForm);
 addCardFormValidator.enableValidation();
+const avatarFormValidator = new FormValidator(config, avatarForm);
+avatarFormValidator.enableValidation();
 
-// Event Listeners
-profileEditBtn.addEventListener("click", () => {
-  const userData = userInfo.getUserInfo();
-  profileTitleInput.value = userData.name;
-  profileDescriptionInput.value = userData.about;
-  profileEditPopup.open();
+document.addEventListener("DOMContentLoaded", () => {
+  const formElement = document.querySelector("#changeprofile_form");
+  const formValidator = new FormValidator(config, formElement);
+  formValidator.enableValidation();
```

Comment:
Please, delete the redundant validator from `index.js`. You have already created all 3 validators